content/javascript/concepts/arrays/arrays.md

```diff
@@ -24,25 +24,23 @@ const mixedArray = [1, 'chicken', false];
 
 ## Creating an Array
 
-Using an array literal is the easiest way to create a JavaScript array.
+The most straightforward way to create an array is using an array literal.
 
 ```js
-var cars = ['Saab', 'Volvo', 'BMW'];
+const fruit = ['Apple', 'Orange', 'Banana'];
 ```
 
-Spaces and line breaks are not important. A declaration can span multiple lines:
-
-```js
-var cars = ['Saab', 'Volvo', 'BMW'];
-```
+Whitespace is ignored and declarations can span multiple lines.
 
 Using the JavaScript Keyword `new` is another way to create an array.
 
 ```js
-var cars = new Array('Saab', 'Volvo', 'BMW');
+const fruit = new Array('Apple', 'Orange', 'Banana');
 ```
 
-## Access the Elements of an Array
+Though, because the effect is exactly the same, it is preferable to use the literal method for simplicity and execution speed.
+
+## Accessing the Elements of an Array
 
 JavaScript array elements are arranged by index values, starting at 0:
```

Comment:
```suggestion
Array elements are ordered by index values, starting at 0:
```

By this point in entry, we would hope the reader is aware they are learning about _JavaScript_ arrays. A few other points with tis section:

- For the last list item, I suggest changing to "Index n has the nth element."
- I would consider changing the `pseudo` code snippet into a `js` code snippet that defines `array` and `console.log`s a valid `index.`